src/main/scala/akka/persistence/cassandra/ConfigSessionProvider.scala

```diff
@@ -110,6 +111,7 @@ class ConfigSessionProvider(system: ActorSystem, config: Config) extends Session
         .withPoolingOptions(poolingOptions)
         .withReconnectionPolicy(new ExponentialReconnectionPolicy(1000, reconnectMaxDelay.toMillis))
         .withQueryOptions(new QueryOptions().setFetchSize(fetchSize))
+        .withPort(port)
```

Comment:
This works. I have verified with [ccm](https://github.com/pcmanus/ccm) with custom ports (first reproduced the issue and then when adding `withPort` it works).

For the record, it's a bit difficult to use ccm with custom ports. Had to define them like:
```
ccm add node1 -i 127.0.0.1:10011 -t 127.0.0.1:10111 -l 127.0.0.1:10211 -j 7411 -r 8411
ccm add node2 -i 127.0.0.2:10011 -t 127.0.0.2:10111 -l 127.0.0.2:10211 -j 7412 -r 8412
ccm add node3 -i 127.0.0.3:10011 -t 127.0.0.3:10111 -l 127.0.0.3:10211 -j 7413 -r 8413
```